frontends/multi/src/router/welcome.rs

```diff
@@ -116,15 +116,13 @@ pub fn screen(sugarloaf: &mut Sugarloaf) {
 }
 
 #[inline]
-#[cfg(target_os = "macos")]
 fn welcome_content() -> String {
-    format!("Your configuration file will be created in\n{}\n\nTo open settings menu use\n\"Command\" + \",\" (comma)\n\n\n\nMore info in raphamorim.io/rio/docs
-    ", rio_backend::config::config_file_path())
-}
+    #[cfg(target_os = "macos")]
+    let shortcut = "\"Command\" + \",\" (comma)";
 
-#[inline]
-#[cfg(not(target_os = "macos"))]
-fn welcome_content() -> String {
-    format!("Your configuration file will be created in\n{}\n\nTo open settings menu use\n\"Control\" + \"Shift\" + \",\" (comma)\n\n\n\nMore info in raphamorim.io/rio/docs
-    ", rio_backend::config::config_file_path())
+    #[cfg(not(target_os = "macos"))]
+    let shortcut = "\"Control\" + \"Shift\" + \",\" (comma)";
+
+    format!("Your configuration file will be created in\n{}\n\nTo open settings menu use\n{}\n\n\n\nMore info in raphamorim.io/rio/docs
+    ", rio_backend::config::config_file_path().to_str().unwrap(), shortcut)
```

Comment:
I would prefer use `unwrap_or_default` or `unwrap_or_else` in this case to ensure will run